xarray/tests/test_variable.py

```diff
@@ -1544,7 +1544,24 @@ def test_reduce_keep_attrs(self):
         vm = v.mean(keep_attrs=True)
         assert len(vm.attrs) == len(_attrs)
         assert vm.attrs == _attrs
+        
+    def test_binary_ops_keep_attrs(self):
+        _attrs = {'units': 'test', 'long_name': 'testing'}
+
+        a = Variable(['x', 'y'], np.random.randn(3,3), _attrs)
+        b = Variable(['x', 'y'], np.random.randn(3,3), _attrs)
 
+        # Test dropped attrs
+        d = a - b   # just one operation
+        assert len(d.attrs) == 0
```

Comment:
I think this is unnecessary given the next line.

Reply:
Agreed

Reply:
Hey, I mean that you could delete the line "assert len(d.attrs) == 0" since you're comparing the attrs dictionary in the next line.